tests/x11regressions/evolution/evolution_meeting_imap.pm

```diff
@@ -0,0 +1,375 @@
+    send_key "ret";
+    save_screenshot;
+    send_key_until_needlematch "evolution_wizard-sending-smtp", "down", 5, 3;
+    wait_screen_change {
+        send_key "ret";
+    };
+    wait_screen_change {
+        send_key "alt-s";
+    };
+    type_string "$mail_server";
+    wait_screen_change {
+        send_key "alt-p";
+    };
+    type_string "$mail_sendport";
+    wait_screen_change {
+        send_key "alt-v";
+    };
+    wait_screen_change {
+        send_key "alt-m";
+    };
+    send_key "ret";
+    send_key_until_needlematch "evolution_wizard-sending-starttls", "down", 5, 3;
+    send_key "ret";
+
+    #Known issue: hot key 'alt-y' doesn't work
+    #wait_screen_change {
+    #   send_key "alt-y";
+    #};
+    #send_key "ret";
+    #send_key_until_needlematch "evolution_wizard-sending-authtype", "down", 5, 3;
+    #send_key "ret";
+    #Workaround of above issue: click the 'Check' button
+    assert_and_click "evolution_wizard-sending-setauthtype";
+    send_key_until_needlematch "evolution_wizard-sending-authtype", "down", 5, 3;
+    send_key "ret";
+    wait_screen_change {
+        send_key "alt-n";
+    };
+    sleep 1;
+    type_string "$mail_user";
+
+    send_key "$next";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "ret";    #only in sp2, send ret to next page
+    }
+    assert_screen "evolution_wizard-account-summary";
+    send_key "$next";
+    if (sle_version_at_least('12-SP2')) {
+        send_key "alt-n";    #only in sp2
+        send_key "ret";
+    }
+    assert_screen "evolution_wizard-done";
+    send_key "alt-a";
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-a";    #disable keyring option, only in SP2
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    if (check_screen "evolution_mail-init-window") {
+        send_key "super-up";
+    }
+    if (check_screen "evolution_mail-auth") {
+        if (sle_version_at_least('12-SP2')) {
+            send_key "alt-p";
+        }
+        type_string "$mail_passwd";
+        send_key "ret";
+    }
+    assert_screen "evolution_mail-max-window";
+}
+
+#Setup mail account by auto lookup
+sub auto_setup {
+    my ($self, $i) = @_;
+    my $config        = $self->getconfig_emailaccount;
+    my $mail_box      = $config->{$i}->{mailbox};
+    my $mail_server   = $config->{$i}->{sendServer};
+    my $mail_user     = $config->{$i}->{user};
+    my $mail_passwd   = $config->{$i}->{passwd};
+    my $mail_sendport = $config->{$i}->{sendport};
+    my $mail_recvport = $config->{$i}->{recvport};
+    my $next          = "alt-o";
+    if (sle_version_at_least('12-SP2')) {
+        $next = "alt-n";
+    }
+
+    # Clean and Start Evolution
+    x11_start_program("xterm -e \"killall -9 evolution; find ~ -name evolution | xargs rm -rf;\"");
+    x11_start_program("evolution");
+    if (check_screen "evolution-default-client-ask") {
```

Comment:
this will work, but if you want the test to move quicker, you can do 
```
assert_screen qw/evolution-default-client-ask test-evolution-1/;
if (match_has_tag 'evolution-default-client-ask') {
   assert_and_click "evolution-default-client-agree";
   assert_screen "test-evolution-1";
}

You see in the SP2 video neatly the extra 30 seconds it was waiting here.